iina/AppDelegate.swift

```diff
@@ -330,6 +330,10 @@ class AppDelegate: NSObject, NSApplicationDelegate, SPUUpdaterDelegate {
       }
 
       if let pc = lastPlayerCore {
+        if commandLineStatus.shufflePlaylist {
+          pc.toggleShuffle()
```

Comment:
Unfortunately, this does not work. Apparently it is being executed too late. Running this over and over again:
```text
iina-cli --mpv-shuffle ~/Movies/
```
Starts playing the exact same file each time. The same file that is played without the `--mpv-shuffle` option. IINA needs to load the playlist, send the shuffle command and then start playing the first file in the playlist. Something like that so the first file played is random. From the behavior I expect the first file is selected before the playlist has been shuffled.